### PR TITLE
papr: Update to later versions

### PR DESCRIPTION
**Pull request by ashcrow**
- Update to version 3.7.0
- Use Fedora 26
- Work around for https://bugzilla.redhat.com/show_bug.cgi?id=1483553

### CODE REVIEW

**Comment by ashcrow** [comment]
Odd. This change should have no impact on ``tox`` or any of the openshift CI tests...

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by sdodson** [comment]
Don't worry about the normal CI process, if fedora/26/atomic starts passing we'll merge.

**Comment by sdodson** [comment]
s/if/when/ :-)

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
The ``registry`` and ``image`` variables were not being defined. Rebased and trying again.

```
  1. Hosts:    ocp-node1, ocp-node2
     Play:     Verify Requirements
     Task:     openshift_health_check
     Message:  One or more checks failed
     Details:  check "docker_image_availability":
               One or more required Docker images are not available:
                   openshift/node:v3.7.0,
                   openshift/openvswitch:v3.7.0,
                   openshift/origin-deployer:v3.7.0,
                   openshift/origin-docker-registry:v3.7.0,
                   openshift/origin-haproxy-router:v3.7.0,
                   openshift/origin-pod:v3.7.0
               Configured registries: docker.io
               Checked by: timeout 10 skopeo inspect --tls-verify=false docker://{registry}/{image}

  2. Hosts:    ocp-master
     Play:     Verify Requirements
     Task:     openshift_health_check
     Message:  One or more checks failed
     Details:  check "docker_image_availability":
               One or more required Docker images are not available:
                   openshift/node:v3.7.0,
                   openshift/openvswitch:v3.7.0,
                   openshift/origin-deployer:v3.7.0,
                   openshift/origin-docker-registry:v3.7.0,
                   openshift/origin-haproxy-router:v3.7.0,
                   openshift/origin-pod:v3.7.0,
                   openshift/origin:v3.7.0
               Configured registries: docker.io
               Checked by: timeout 10 skopeo inspect --tls-verify=false docker://{registry}/{image}
```

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
@sdodson even though this is failing I believe it's ok to merge. The failure is that the early tests can't find the images and it seems like something outside of ``papr`` is no longer setting registry and image. causing the failure.

**Comment by sdodson** [comment]
bot, retest this please

**Comment by sdodson** [comment]
Switched you to a tag that exists, see https://hub.docker.com/r/openshift/origin/tags/ for tags

**Comment by sdodson** [comment]
Did we need to use 3.7 builds for some reason? I'd prefer sticking with a GA build if possible, it's hard to debug if these problems are 3.7.0.alpha.1 related or installer problems.

**Comment by ashcrow** [comment]
We can bring it back to 3.6.

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
```
Sep 26 20:37:49 ocp-master systemd[1]: systemd-coredump@1-8237-0.service: Failed to fork: Cannot allocate memory
Sep 26 20:37:49 ocp-master systemd[1]: systemd-coredump@1-8237-0.service: Failed to run 'start' task: Cannot allocate memory
```

Resource starved.

**Comment by ashcrow** [comment]
Opened up https://github.com/projectatomic/papr/issues/63 in relation to this.

File: .papr.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@
 #   https://github.com/projectatomic/papr/blob/master/sample.papr.yml
 
 cluster:
+  specs:
+      ram: 4096
```
Comment (jlebon) [inline .papr.yml]: Note the `specs` key is per host (e.g. you could have different specs for each host), so you need to repeat it three times for each item in the `hosts` list below. Might make sense to also accept a global cluster-level `specs` key as well.

**Comment by ashcrow** [comment]
@sdodson We have a working ``papr`` setup once again!

**Comment by jlebon** [comment]
@ashcrow Awesome! Thanks a lot for this! 👍

**Comment by jlebon** [comment]
Minor follow-up to this in https://github.com/openshift/openshift-ansible/pull/5461.

**Comment by jlebon** [comment]
And of course, I actually meant https://github.com/openshift/openshift-ansible/pull/5561. (Though cool they're exactly offset by 100!).

**Pull request merged**